docs/jade-getting-started.md

```diff
@@ -318,7 +316,7 @@ the above test commands.  More specific examples are available in
 
 ### 5. Build `jade-data-repo-ui`
 
-Follow the [setup instructions](https://github.com/DataBiosphere/jade-data-repo-ui#jade-data-repository-ui) to build the `jade-data-repo-ui` repository.
+To do API and UI development simultaneously, Follow the [setup instructions](https://github.com/DataBiosphere/jade-data-repo-ui#jade-data-repository-ui) to build the `jade-data-repo-ui` repository.
 
 By setting the `PROXY_URL` environment variable, you can point the UI to your local data repo instance.
```

Comment:
I wasn't sure if I should just remove how-tos for jade-data-repo-ui altogether, and let that be that repo's problem. That was my inclination, but I wanted to check with yall first.

Reply:
Perhaps leave those instructions out but make sure we retain the reference to `jade-data-repo-ui`